lib/manager/helm-requirements/extract.ts

```diff
@@ -50,6 +51,12 @@ export async function extractPackageFile(
     if (dep.repository) {
       res.registryUrls = [dep.repository];
       if (dep.repository.startsWith('@')) {
+        const alias = config.aliases[dep.repository.slice(1)];
```

Comment:
Can we either add a documentation line describing the `slice(1)` or perhaps split it into two lines with a meaningful variable name if that's better?